### ISSUE TITLE
Investigate localization

### ISSUE THREAD

**Issue by blowfishpro** [KSP-RO/AJE#9]

**Comment by blowfishpro** [comment]
@duck1998 I noticed you moved some files within your localized version of AJE.  It's making it hard to match things up.  Is there a particular reason you did this?

**Comment by blowfishpro** [comment]
Ahh, I see you organized it as a bunch of additional patches.  Might be able to do something with that.  Could take some time though, since that's not the format KSP expects localization to be in.

**Comment by Duck1998** [comment]
Yes, that is for working with ModuleManager. I can help with the KSP's format.

**Comment by blowfishpro** [comment]
I sat down and wrote a script to match things up, just needs a few more changes which I will make tomorrow.

**Comment by blowfishpro** [comment]
I've done the localization here: https://github.com/KSP-RO/AJE/tree/Localization

The Chinese localization table is here: https://github.com/KSP-RO/AJE/blob/94c1760430d26f21388ac15990c224372f24c9a8/GameData/AJE/Localization/cz.cfg - there were a few things without localization in your version, mostly toward the bottom, I left them at the English versions but feel free to submit corrections.

Also let me know if there's anything I missed.

**Comment by blowfishpro** [comment]
One thing I still need to do is go through the actual code and extract out all localizable content from that.

**Comment by Duck1998** [comment]
Missing translations updated. You can check it in pull requests.
